src/ontology/uberon-edit.obo

```diff
@@ -190993,8 +190990,8 @@ name: rima vulvae
 def: "The fissure between the labia majora" [https://www.thefreedictionary.com/pudendal+cleft]
 synonym: "pudendal cleft" EXACT [FMA:19995]
 synonym: "pudendal cleft" EXACT DEPRECATED [doi:10.1002/ca.23705]
-synonym: "rima pudendi" EXACT DEPRECATED [doi:10.1002/ca.23705]
 synonym: "rima pudendi" EXACT LATIN [A09.2.01.006]
+synonym: "rima pudendi" EXACT DEPRECATED [doi:10.1002/ca.23705]
```

Comment:
not sure how these edits occurred... I did not add and delete this

Reply:
I think this is just a serialisation bug => change of line order.  No change to meaning/content of file, so fine to ignore.